utils/build_swift/argparse/types.py

```diff
@@ -172,7 +172,7 @@ def __init__(self):
 
     def __call__(self, value):
         matches = super(ClangVersionType, self).__call__(value)
-        components = filter(lambda x: x is not None, matches.group(1, 2, 3, 5))
+        components = list(filter(lambda x: x is not None, matches.group(1, 2, 3, 5)))
```

Comment:
In Python I normally like more the list comprehensions than the functions.

```python
components = [x for x in matches.group(1, 2, 3, 5) if x is not None]
```

Reply:
Another case of learning as I went :)